### ISSUE TITLE
The style of toggle button is in conflict with the css of my application

### ISSUE THREAD

**Issue by david-fernando** [longyarnz/shared-ui#7]
The problem is with the "button {...}" and "* {..}" in your css file
Link of component: https://bit.dev/lekanmedia/shared-ui/toggle-button

**Comment by longyarnz** [comment]
Yeah, that's an issue. I wrote the library a while back. I'll refactor the component to use styled-components instead. That should help scope the style.
